<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecipe</class>
 <widget class="QDialog" name="AddRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>561</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="recipeNameLabel">
      <property name="text">
       <string>Recipe Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="recipeNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ingredientsLabel">
      <property name="text">
       <string>Ingredients</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ingredientsLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cookingTimeLabel">
      <property name="text">
       <string>Cooking Time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cookingTimeLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="nutritionLabel">
      <property name="text">
       <string>Nutrition</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="nutritionLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="instructionLabel">
      <property name="text">
       <string>Instruction</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="txtInstruction"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
